### PR TITLE
Mhv account params for registration and validations

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

File: lib/mhv_ac/registration_form.rb [hunk 1]
```diff
@@ -33,9 +33,11 @@ class RegistrationForm < Common::Form
     attribute :terms_version, String
     attribute :terms_accepted_date, Common::HTTPDate
 
-    validates :icn, :is_patient, :is_veteran, :email, presence: true
-    validates :address1, :city, :state, :zip, :country, presence: true
-    validates :sign_in_partners, :terms_version, :terms_accepted_date, presence: true
+    # NOTE: commenting out these validations because it's not entirely clear they're required
+    # instead for now we're going to rely on MHV kicking back its error based on these.
+    # validates :icn, :is_patient, :is_veteran, :email, presence: true
```
Comment (patrickvinograd) [inline lib/mhv_ac/registration_form.rb]: I think we should keep some of these in place - icn, email, is_patient, is_veteran, sign_in_partners, terms_version, terms_accepted_date are all either provided/derived by us or guaranteed to be present.

Comment (patrickvinograd) [inline lib/mhv_ac/registration_form.rb]: But if you are just removing them all for testing purposes that's cool.

Comment (saneshark) [inline lib/mhv_ac/registration_form.rb]: yea, so if they're required by MHV they will raise an error. I don't know what good it does for us pre-emptively raising that error unless theirs are just completely nonsensical. If it results in a BackendServiceException we can always handle it. I think these type of validations are particularly useful if you want to validate user input, maybe a little less so if everything is derived by vets.gov on the backend.

**Comment by saneshark** [comment]
@patrickvinograd I think is good to merge, we can uncomment out those validations if we find that address information errors returned by MHV are not possible to map to more human readable user friendly error on our end. Waiting for the LGTM or for you to just merge when you're ready.

**Pull request merged**